README.md

```diff
@@ -135,9 +135,9 @@ const main = async () => {
   if (!('IdleDetector' in window)) {
     return console.log('IdleDetector is not available.');
   }
-  // Check if notifications permission is granted.
-  if ((await navigator.permissions.query({name: 'notifications'})).state !== 'granted') {
-    return console.log('Notifications permission not granted.');
+  // Check if permission is granted.
+  if ((await navigator.permissions.query({name: 'idle-detection'})).state !== 'granted') {
```

Comment:
ah, interesting. is this a completely new permission? or can it be bundled with others (e.g. the notifications one)? or is this meant to be a choice that implementations can make?